images/rook/Dockerfile.base

```diff
@@ -14,48 +14,32 @@
 
 FROM BASEIMAGE
 
-ARG EXTRALIBS
-
-# install runtime dependencies for ceph and rook
-RUN BOOST_VERSION=1.62.0 && \
+# add Ceph and other packages needed
+ADD 'https://download.ceph.com/keys/release.asc' /tmp/release.asc
+RUN CEPH_VERSION=luminous && \
+    apt-key add /tmp/release.asc && \
+    echo "deb http://download.ceph.com/debian-$CEPH_VERSION/ xenial main" | tee /etc/apt/sources.list.d/ceph-$CEPH_VERSION.list && \
```

Comment:
This will pull the latest luminous release? So we might pick up a new luminous build in a dot release?

Reply:
Yes see thr discussion in this PR. I think its a reasonable tradeoff to make at this point. We still have the manual ability to revert if something went wrong